package.json

```diff
@@ -1,81 +1,88 @@
-    "rollup-plugin-bundle-size": "^1.0.3",
-    "rollup-plugin-dts": "^4.1.0",
-    "rollup-plugin-peer-deps-external": "^2.2.4",
-    "rollup-plugin-postcss": "^4.0.2",
-    "rollup-plugin-svg": "^2.0.0",
-    "rollup-plugin-terser": "^7.0.2",
-    "styled-components": "^5.3.3",
-    "stylelint": "^14.4.0",
-    "stylelint-config-recommended": "^7.0.0",
-    "stylelint-config-styled-components": "^0.1.1",
-    "stylelint-processor-styled-components": "^1.10.0",
-    "typescript": "^4.5.5"
-  },
-  "peerDependencies": {
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "react-paginate": "^8.1.3",
-    "styled-components": "^5.3.3"
-  },
-  "resolutions": {
-    "@types/react": "17.0.2",
-    "@types/react-dom": "17.0.2"
-  },
-  "dependencies": {
-    "polished": "^4.1.4",
-    "react-circle-flags": "^0.0.17",
-    "react-circular-progressbar": "^2.1.0",
-    "react-tabs": "^4.2.0",
-    "react-toastify": "^8.2.0",
-    "styled-gen": "^1.4.8",
-    "styled-reset": "^4.3.4"
-  }
+    "name": "@impact-market/ui",
+    "version": "1.5.6",
+    "engines": {
+        "node": ">=18"
+    },
+    "main": "dist/cjs/index.js",
+    "module": "dist/esm/index.js",
+    "types": "dist/esm/index.js",
+    "license": "MIT",
+    "scripts": {
+        "build": "rollup -c",
+        "generate:changelog": "auto-changelog -p",
+        "generate:icons": "yarn svglist generate",
+        "lint": "eslint .",
+        "release": "tasks/release.sh",
+        "storybook": "NODE_OPTIONS=--openssl-legacy-provider start-storybook -p 6006",
+        "build-storybook": "NODE_OPTIONS=--openssl-legacy-provider build-storybook"
+    },
+    "dependencies": {
+        "@storybook/manager-webpack5": "6.5.16",
+        "polished": "4.2.2",
+        "react-circle-flags": "0.0.18",
+        "react-circular-progressbar": "2.1.0",
+        "react-tabs": "4.2.0",
+        "react-toastify": "8.2.0",
+        "styled-gen": "1.4.8",
+        "styled-reset": "4.4.7"
+    },
+    "devDependencies": {
+        "@babel/core": "7.17.2",
+        "@psoares/svg-list": "1.1.10",
+        "@rollup/plugin-commonjs": "21.0.2",
+        "@rollup/plugin-node-resolve": "13.1.3",
+        "@rollup/plugin-typescript": "8.3.0",
+        "@storybook/addon-actions": "6.5.16",
+        "@storybook/addon-essentials": "6.5.16",
+        "@storybook/addon-links": "6.5.16",
+        "@storybook/addon-postcss": "2.0.0",
+        "@storybook/addons": "6.5.16",
+        "@storybook/react": "6.5.16",
+        "@storybook/theming": "6.5.16",
+        "@types/lodash": "4.14.195",
+        "@types/react": "17.0.39",
+        "@types/react-dom": "17.0.14",
+        "@types/react-tabs": "2.3.4",
+        "@types/styled-components": "5.1.26",
+        "@typescript-eslint/eslint-plugin": "5.11.0",
+        "auto-changelog": "2.4.0",
+        "autoprefixer": "10.4.14",
+        "babel-loader": "8.2.3",
+        "eslint": "8.40.0",
+        "eslint-config-impact-market": "impactMarket/eslint-config-impact-market",
```

Comment:
you can bump this to the actual release, 2.2.0

Reply:
what do you mean? change the UI version in package.json from 1.5.6 to 2.2.0? (and bump it also as 2.2.0)

Reply:
no, the `eslint-config-impact-market`